_announcements/01_apple_common_removal.md

```diff
@@ -0,0 +1,319 @@
+2. Double-click on the file.
+3. When prompted, enter your password to install the certificate.
+
+Non-administrative users may follow the steps above to install COMMON in the Login Keychain, which is specific to their accounts. This will not impact other user accounts on a device. 
+
+### iOS&nbsp;&mdash;&nbsp;Install COMMON Options
+You can install COMMON in your iOS device's Trust Store by using one of these options:
+
+#### Option 1:&nbsp;&nbsp;Install COMMON Using Safari Web Browser
+1. Launch Safari.
+2. Navigate to the [COMMON root CA certificate](http://http.fpki.gov/fcpca/fcpca.crt){:target="_blank"}<br>
+> System message appears: *The website is trying to open Settings to show you a configuration profile. Do you want to allow this?*<br>
+3. Click *Allow*.<br>
+> The COMMON Configuration Profile appears.<br> 
+4. Click *More Details* and then the COMMON certificate entry.
+5. Scroll down to *Fingerprints* and verify the certificate's SHA-256 hash.
+6. At top left of screen, click *Back* and *Install Profile*. Then, click *Install* (top right).
+7. When prompted, enter your device passcode.
+8. Click *Install* (top right), and *Install* again.
+9. Click *Done*.
+
+Next, you'll need to enable full trust for COMMON:
+
+1. From the iOS device's *Home* screen, go to *Settings* -> *General* -> *About* -> *Certificate Trust Settings*.
+2. Beneath *Enable Full Trust for Root Certificates*, toggle _ON_ for the Federal Common Policy CA entry. 
+3. When certificate appears, click *Continue*.
+4. You can now successfully navigate to an intranet site whose TLS certificate was issued by a Federal PKI CA.
+
+
+#### Option 2:&nbsp;&nbsp;Install COMMON Using Apple Configuration Profile
+You can use the macOS configuration profile procedures above to install COMMON on iOS devices: [Apple Configuration Profiles](#install-using-an-apple-configuration-profile).
+
+
+### tvOS&nbsp;&mdash;&nbsp;Install COMMON Options
+If you would like to install COMMON on tvOS government-furnished equipment, email us at fpki@gsa.gov.
+
+## Frequently Asked Questions
+
+### 1.&nbsp;&nbsp;Why is COMMON being removed? 
+The Federal PKI CAs don't comply with Apple's requirements for globally trusted TLS/SSL certificates. Apple's requirements include: 
+
+**a.&nbsp;&nbsp;Requirement for Fully-Qualified Domain Names (FQDNs)**<br>
+Apple plans to restrict TLS/SSL certificates to only those certificates using FQDNs ending in .gov, .mil, or fed.us. Some federal agencies issue TLS/SSL certificates to intranet assets. These certificates either:&nbsp;&nbsp;don't have FQDNs; contain intranet domains that don't end in .gov, .mil, or fed.us; or use short names (aliases). Under Apple's requirements, these agencies would need to reissue, re-install, and reconfigure all "non-compliant" certificates and applications. The Federal PKI community has determined that this would have a negative impact on mission applications on the intranets.
+
+**b.&nbsp;&nbsp;Requirement for public audit**<br>
```

Comment:
Update or remove?  Apple requires webtrust audits, and public documents.  It's also not related to the .gov / .mil domain constraint.

Reply:
Removed.